### ISSUE TITLE
Package WebWindow.Blazor 0.1.0-20200214.10 is not compatible with netstandard2.1

### ISSUE THREAD

**Issue by LeonardoEstrela99** [SteveSandersonMS/WebWindow#115]
I'm getting this error while trying to install the package on my Blazor WebAssembly project. Is the project outdated? Can I do something to make it work on my project?

**Comment by eivarin** [comment]
yes u can. you will need to at least import the webwindow.blazor project and if you want to you can use the webwindow nugget package on that project so that u don't have to import the full library. then just manually update the versions of blazor on the webwindow.blazor project. now, on the main  blazor wasm project, just import the webwindow.blazor dll and make sure to remove the references of the nugget blazor packages there.

To start up the app you will need to make a start up class inside the program.cs
It should look something like this
![image](https://user-images.githubusercontent.com/48986364/88115254-a5395480-cbad-11ea-974b-cb65057c9ec7.png)
ignore the error on the run<startup> its there because I modified one of my programs and just made it simple enough to put here
